lib/worker/app.js

```diff
@@ -3,10 +3,17 @@
 //        so that method is more easily accessible.
 
 const { Probot } = require('probot');
-const { getPrivateKey } = require('@probot/get-private-key');
+const { AppSecrets , Installation} = require('../models');
 
-module.exports = new Probot({
-  appId: process.env.APP_ID,
-  privateKey: getPrivateKey(),
-  logLevel: process.env.LOG_LEVEL,
-});
+
+module.exports = async (jiraHost) => { 
+  const installation = await Installation.getForHost(jiraHost);
+  const host = installation.githubHost;  
+  const ghaeInstanceData = await AppSecrets.getForHost(host); 
```

Comment:
add check for "ghaeInstanceData", if this is empty / null log and return, even for other case as well add the trace statement